tfjs-backend-webgpu/src/index.ts

```diff
@@ -37,11 +37,17 @@ if (isWebGPUSupported()) {
     };
 
     const adapter = await navigator.gpu.requestAdapter(gpuDescriptor);
-    let deviceDescriptor: GPUDeviceDescriptor = {};
+    const adapterLimits = adapter.limits;
+    const deviceDescriptor: GPUDeviceDescriptor = {};
     const supportTimeQuery = adapter.features.has('timestamp-query');
+    deviceDescriptor.requiredLimits = {
+      'maxComputeWorkgroupStorageSize':
+          adapterLimits.maxComputeWorkgroupStorageSize,
+      'maxStorageBufferBindingSize': adapterLimits.maxStorageBufferBindingSize,
+    };
```

Comment:
Only two limits are initialized here,  and we declare 3. So an iteration of dict can avoid such kind of mistake.